.github/workflows/smp.yml

```diff
@@ -49,11 +49,11 @@ jobs:
         make
         ./charmrun +p1 ./Gravity -f ../inputgen/1k.tipsy ++ppn 1 +setcpuaffinity ++local -u 2
         ./charmrun +p1 ./Gravity -f ../inputgen/1k.tipsy ++ppn 1 +setcpuaffinity ++local -u 2 -n 1 -p 1 -d sfc -t kd
-        ./charmrun +p4 ./Gravity -f ../inputgen/1k.tipsy ++ppn 4 +setcpuaffinity ++local -u 2 -t bin
+        ./charmrun +p4 ./Gravity -f ../inputgen/1k.tipsy ++ppn 4 +setcpuaffinity ++local -u 2 -t binoct
```

Comment:
Where the previous arg-parser would allow invalid values, the current one will fail here.

Reply:
That is an acceptable outcome.